packages/docker-node/src/ckb-bin-network.ts

```diff
@@ -0,0 +1,259 @@
+import { Address, CellDep, Indexer, RPC, Transaction, commons, config, hd, helpers } from '@ckb-lumos/lumos'
+import { CKBNode } from './interface'
+import type { BinNodeStartOptions, BinNodeStopOptions, DeployOptions, InfraScript } from './types'
+import { spawn, execSync } from 'child_process'
+import { generateDevConfig } from './helper'
+import fs from 'fs'
+import path from 'node:path'
+import { waitUntilCommitted } from '@ckb-js/kuai-common'
+
+export class CKBBinNetwork implements CKBNode {
+  #port = '8114'
+  #host = '127.0.0.1'
+  #lumosConfig: config.Config = config.getConfig()
+
+  stop({ ckbPath, clear }: BinNodeStopOptions): void {
+    if (fs.existsSync(path.resolve(ckbPath, 'pid'))) {
+      const indexer = fs.readFileSync(path.resolve(ckbPath, 'pid', 'indexer'), 'utf-8')
+      spawn('kill', ['-9', indexer])
```

Comment:
We'd better not stop ckb by `kill -9`, it may break the ckb data file, then it cannot start next time.
It's better to `kill 15` means `kill with SIGTERM`. https://zh.wikipedia.org/wiki/Unix%E4%BF%A1%E5%8F%B7#SIGTERM